### PR TITLE
OCPQE-22292 fix aws-ipi-efa-pg installation fail issue

### PR DESCRIPTION
**Pull request by huali9**
@yunjiang29 @sunzhaohua2 @miyadav @shellyyang1989 PTAL, thanks!

### CODE REVIEW

**Comment by huali9** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-aws-ipi-efa-pg-f5

**Comment by huali9** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-aws-ipi-efa-pg-f5

**Comment by huali9** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-aws-ipi-efa-pg-f5

**Comment by huali9** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-aws-ipi-efa-pg-f5

**Comment by huali9** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-aws-ipi-efa-pg-f5

**Comment by huali9** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-aws-ipi-efa-pg-f5

**Comment by huali9** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-aws-ipi-efa-pg-f5

**Comment by huali9** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-aws-ipi-efa-pg-f5

**Comment by huali9** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-aws-ipi-efa-pg-f5

**Comment by huali9** [comment]
The installation pass now. https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/53218/rehearse-53218-periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-aws-ipi-efa-pg-f5/1802665369115037696

File: ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh [hunk 1]
```diff
@@ -482,24 +482,25 @@ EOF
 function enable_efa_pg_instance_config() {
```
Comment (yunjiang29) [inline ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh:482]: I mean remove `enable_efa_pg_instance_config` totally from `ipi-install-install`, but using `ipi-install-install-aws` instead, both `jq` and `yq` are available in that image.

Comment (huali9) [reply ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh:482]: I got your meaning before, and I tried that, but met some issue, and I have no idea how to solve it.
https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/53218/rehearse-53218-periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-aws-ipi-efa-pg-f5/1802612273714302976 
https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/53218/rehearse-53218-periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-aws-ipi-efa-pg-f5/1802594765452087296
By the way, there is no `yq` in that image, got error "/bin/bash: line 250: yq: command not found" in https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/53218/rehearse-53218-periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-aws-ipi-efa-pg-f5/1802575797823737856
Another, I think updating in `ipi-install-install` should be ok, there is similar code in it. https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh#L455

Comment (huali9) [reply ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh:482]: @yunjiang29 Does the updating in `ipi-install-install` look good to you? Or do you insist on updating in `ipi-install-install-aws`? If yes, can you please take a look at the error, it doesn't look something I updated.

Comment (yunjiang29) [reply ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh:482]: > I got your meaning before, and I tried that, but met some issue, and I have no idea why it fail.

@huali9 As the commit is gone, I can't see the changes, but as your changes is very simple, I don't think it's caused  by your changes, can you update again and re-trigger the job?

>it has yq-go instead.

yes, `yq-go` is the correct command, it's version 3.3 of yq.

> updating in ipi-install-install-aws

The reason is this step is applicable on AWS platform, ipi-install-install-aws is more proper for this.

**Comment by huali9** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-aws-ipi-efa-pg-f5

**Comment by huali9** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-aws-ipi-efa-pg-f5

**Comment by huali9** [comment]
The installation passed https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/53218/rehearse-53218-periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-aws-ipi-efa-pg-f5/1803331367619006464 
@yunjiang29 PTAL again, thanks!

**Comment by yunjiang29** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.15-e2e-aws-ovn-edge-zones

**Pull request merged**